### DISCUSSION TITLE
How to delete css-loader/dist/cjs.js comments

### DISCUSSION THREAD

**Discussion by woody-li** [webpack/webpack#16616] [Q&A] [answered]
Using `mini-css-extract-plugin` and `css(less)-loader` to build css file from a css/less entry.
There's a comment at dist file header, and more comments in less entry.
```
css ./node_modules/css-loader/dist/cjs.js!./pack.css
```
```
css ./node_modules/css-loader/dist/cjs.js!./node_modules/less-loader/dist/cjs.js??ruleSet[1].rules[3].use[3]!./pack.less
```

Is it possibile to prevent this comment?

**Comment by alexander-akait** [comment, marked as answer]
Please set https://webpack.js.org/configuration/output/#outputpathinfo to `false`

**Comment by woody-li** [comment]
Thanks for your reply, it resolves my problem.
